Fortinet/fortigate/ingest/parser.yml

```
@@ -203,6 +203,8 @@ stages:
           http.request.method: "{{parsed_event.message.method}}"
           log.level: "{{parsed_event.message.level or parsed_event.message.FTNTFGTlevel or parsed_event.message.FortinetFortiGatelevel}}"
           observer.hostname: "{{parsed_event.message.devname}}"
+          host.name: "{{parsed_event.message.srcname}}"
+          os.family: "{{parsed_event.message.osname}}"
```

`os` field must be nested in `host`
```suggestion
          host.os.family: "{{parsed_event.message.osname}}"
```